content/docs/quic/flowcontrol.md

```diff
@@ -44,7 +44,7 @@ quic.Config{
 }
 ```
 
-The initial limits (`InitialStreamReceiveWindow` and `InitialConnectionReceiveWindow`) are advertised to the peer during the QUIC handshake, and apply to every new stream opened by the peer. The protocol doesn't provide a way to change these limits after completion of the handshake. 
+The initial limits (`InitialStreamReceiveWindow` and `InitialConnectionReceiveWindow`) are advertised to the peer during the QUIC handshake, and apply to every new stream opened by the peer. The protocol doesn't provide a way to change these limits after the completion of the handshake. 
```

Comment:
ChatGPT claims that both are correct, and I think I prefer the original version.

```suggestion
The initial limits (`InitialStreamReceiveWindow` and `InitialConnectionReceiveWindow`) are advertised to the peer during the QUIC handshake, and apply to every new stream opened by the peer. The protocol doesn't provide a way to change these limits after completion of the handshake. 
```